packages/sdk/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@pipcook/sdk",
   "version": "1.0.4",
   "description": "JavaScript SDK for accessing Pipcook Daemon",
-  "main": "dist/index.js",
+  "main": "src/index.ts",
```

Comment:
Why do we have this change?

Reply:
It's should not be committed, I will roll it back.